tsl/src/data_node.c

```diff
@@ -1123,6 +1126,16 @@ data_node_modify_hypertable_data_nodes(const char *node_name, List *hypertable_d
 								 "space partitions was set to match the number of data nodes.")));
 				}
 			}
+
+			if (op_type == OP_DETACH && drop_hypertable)
+			{
+				/* Drop the hypertable on the data node */
+				ts_dist_cmd_run_on_data_nodes(psprintf("DROP TABLE IF EXISTS %s.%s",
+													   NameStr(ht->fd.schema_name),
+													   NameStr(ht->fd.table_name)),
```

Comment:
```suggestion
				ts_dist_cmd_run_on_data_nodes(psprintf("DROP TABLE IF EXISTS %s",
													   quote_qualified_identifier(NameStr(ht->fd.schema_name),
													   NameStr(ht->fd.table_name))),
```

Reply:
So, I figured this wasn't necessary since the data comes from our own internal metadata. But I guess it doesn't hurt. But why only quote the schema and not the table name?

Reply:
Fixed.